packages/flutter/lib/src/material/chip.dart

```diff
@@ -94,6 +94,9 @@ abstract interface class ChipAttributes {
 
   /// The color and weight of the chip's outline.
   ///
+  /// If a border is provided in the [shape] property, this property
```

Comment:
This is inconsistent with the ButtonStyleButton combines its shape and border:
```dart
resolvedShape!.copyWith(side: resolvedSide),
```
In other words, if a side parameter is provided it overrides the shape's side (if any).

Reply:
Ohh great catch.  Yes, the `side`parameter should override the shape' side.

My expected result matches. I was confused about that part when both were provided.

Update the PR and the [description](https://github.com/flutter/flutter/pull/132941#issue-1858956224)